### PR TITLE
WandiMailerBundle recipe

### PR DESCRIPTION
**Pull request by jacquesndl**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by jacquesndl** [comment]
Hello, can an admin merge this PR ? No reply since Aug 2. Thanks

File: wandi/mailer-bundle/1.0/config/packages/wandi_mailer.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+wandi_mailer:
+    sender:
+        name: 'example'
```
Comment (maxhelias) [inline wandi/mailer-bundle/1.0/config/packages/wandi_mailer.yaml]: You can also use the env variable for define the name & address like this PR : https://github.com/symfony/recipes-contrib/pull/398/files and then explain the next step in the post-install

Comment (jacquesndl) [reply wandi/mailer-bundle/1.0/config/packages/wandi_mailer.yaml]: @maxhelias thanks for the advice

File: wandi/mailer-bundle/1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Wandi\\MailerBundle\\WandiMailerBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "WANDI_MAILER_SENDER_NAME": "Zinédine Zidane",
+        "WANDI_MAILER_SENDER_ADDRESS": "zinedine.zidane@france.fr"
```
Comment (maxhelias) [inline wandi/mailer-bundle/1.0/manifest.json]: I do not think it's a good idea to set these values ​​by default. Something more generic would be better like `test@test.com`

Comment (Nyholm) [reply wandi/mailer-bundle/1.0/manifest.json]: I agree with Max. Do a more generic example. That will make it more obvious that these values should be changes. 

After that, Im good with merging this PR.

Comment (jacquesndl) [reply wandi/mailer-bundle/1.0/manifest.json]: Ok, it's changed. Thanks.

**Comment by jacquesndl** [comment]
Hello, can an admin merge this PR ? No reply since Nov 14th. Thanks

**Pull request merged**